benchmark/src/index.ts

```diff
@@ -10,7 +10,7 @@ const REPO_PATH = path.join(process.cwd(), REPO_ROOT);
 const REPETITIONS = 5;
 
 const DEFAULT_EXEC_OPTS = { stdio: "ignore" as const, cwd: REPO_PATH };
-const TURBO_BIN = path.resolve(path.join("..", "cli", "turbo"));
+const TURBO_BIN = path.resolve(path.join("..", "target", "debug", "turbo"));
```

Comment:
```suggestion
const TURBO_BIN = path.resolve(path.join("..", "target", "release", "turbo"));
```

Running benchmarks against a debug build is gonna be invalid. I don't know if we have a release build on this task or not.

Reply:
I had to add a `make` target that used `--release`  to get this. Not sure why we don't already have one or if we should be using it everywhere else also, but I agree we should use an optimized build[^1].

[^1]: I haven't looked up what an optimized build is in Rust/cargo is though

Reply:
Note that this script expects turbo to already be built, so the new make target doesn't affect anything else